Nos/Models/Relay+CoreDataClass.swift

```diff
@@ -93,6 +94,13 @@ public class Relay: NosManagedObject {
             return relay
         }
     }
+
+    class func find(supporting nip: Int, for author: Author, context: NSManagedObjectContext) async throws -> [Relay] {
+        try await context.perform {
+            let relays = try context.fetch(Relay.relays(for: author))
+            return relays.filter { $0.metadata?.supportedNIPs?.contains(nip) ?? false }
```

Comment:
There isn't a way to do this filter using a predicate unless we refactor the list of supported nips as a one-to-many relationship that I find too much or a string with ints separated by something. Given that the list of relays shouldn't be too large, I think we can get away with this simple filter.